### PR TITLE
[RELEASES] 2.0.0

### PR DESCRIPTION
**Pull request by jfcalvo**
Release `2.0.0` branch.

### CODE REVIEW

File: argilla/CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,19 @@ These are the section headers that we use:
 * "Security" in case of vulnerabilities.
 -->
 
-## [2.0.0rc1](https://github.com/argilla-io/argilla/compare/v1.29.0...v2.0.0rc)
+## [Unreleased]()
+
+## [2.0.0](https://github.com/argilla-io/argilla/compare/v2.0.0rc1...v2.0.0)
+
+- Added core class refactors. For an overview, see [this blog post](https://argilla.io/blog/introducing-argilla-new-sdk/)
```
Comment (jfcalvo) [inline argilla/CHANGELOG.md:23]: @burtenshaw please can you organize the changes into different sections as we usually do? 

We use the sections defined at https://keepachangelog.com/en/1.1.0

File: argilla-frontend/README.md [hunk 2]
```diff
@@ -31,8 +31,7 @@
 </a>
 </p>
 
-Argilla is a **collaboration platform for AI engineers and domain experts** that require **high-quality outputs, full data ownership, and overall efficiency**.
-
+Argilla is a collaboration tool for AI engineers and domain experts who need to build high-quality datasets for their projects.
 If you just want to get started, we recommend our [UI demo](https://demo.argilla.io/sign-in?auth=ZGVtbzoxMjM0NTY3OA%3D%3D) or our [free Hugging Face Spaces deployment integration](https://huggingface.co/new-space?template=argilla/argilla-template-space). Curious, and want to know more? Read our [documentation](https://argilla-io.github.io/argilla/latest/).
```
Comment (burtenshaw) [inline argilla-frontend/README.md]: @davidberenstein1957 @frascuchon Will these auth links work with the changes to `demo.argilla.io`?

Comment (frascuchon) [reply argilla-frontend/README.md]: Not sure, since the login page is not the same cc @damianpumar

Comment (dvsrepo) [reply argilla-frontend/README.md]: @burtenshaw I'd like to replace this link [free Hugging Face Spaces deployment integration](https://huggingface.co/new-space?template=argilla/argilla-template-space).

with the docs.argilla.io/latest/getting_started/quickstart of the docs. the reasoning is that without context people tends to get confused when they see the space creation wizard

Comment (burtenshaw) [reply argilla-frontend/README.md]: @dvsrepo I understand that people would get confused by the config. Let's change `https://huggingface.co/new-space?template=argilla/argilla-template-space`.

Comment (jfcalvo) [reply argilla-frontend/README.md]: > @davidberenstein1957 @frascuchon Will these auth links work with the changes to demo.argilla.io?

The Cloudflare redirect will respect the path and query string so I guess it should work.

Comment (frascuchon) [reply argilla-frontend/README.md]: I've tested passing the query string and it works. The welcome page allows to sign in with HF Auth

**Pull request merged**